framework/validators/CompareValidator.php

```diff
@@ -162,6 +165,23 @@ public function validateAttribute($model, $attribute)
         }
     }
 
+    /**
+     * @return string
+     */
+    public function getOperator()
+    {
+        return $this->_operator;
+    }
+
+    /**
+     * @param string $operator
+     */
+    public function setOperator($operator)
+    {
+        $this->_operator = $operator;
+        $this->init();
```

Comment:
calling init() multiple times is not expected (e.g. when overriding a class). We should avoid such behavior.

Reply:
Yes. I close this pr

because this is a example